packages/calypso-products/src/plans-list.tsx

```diff
@@ -1591,6 +1591,7 @@ PLANS_LIST[ PLAN_WPCOM_FLEXIBLE ] = {
 };
 
 PLANS_LIST[ PLAN_WPCOM_PRO ] = {
+	...getDotcomPlanDetails(),
```

Comment:
Does this do anything?  It adds `getAnnualPlansOnlyFeatures()` but those don't seem to get checked in `hasFeature()` calls.  (Also, the Pro plan only has annual plans anyway.)

But as far as I can tell, it doesn't cause any harm either.